account_invoice_validate_send_email/models/account_invoice.py

```diff
@@ -18,6 +18,10 @@ class AccountInvoice(models.Model):
     )
     web_url = fields.Char()
 
+    slip_number = fields.Text(
```

Comment:
```suggestion
    carrier_tracking_refs = fields.Char(
```
I think it's OK to keep all the numbers in a single line. Try to follow the naming of the related field in the picking.